### PR TITLE
Foreman birthday celebration

### PR DESCRIPTION
**Pull request by rabajaj0509**

### CODE REVIEW

File: _posts/2019-08-28-foreman-birthday-party-at-netways.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+<p float="left" align='center'>
+  <img src="/static/images/blog_images/2019-birthday-celebration-images/image2.JPG" width="200" />
+  <img src="/static/images/blog_images/2019-birthday-celebration-images/image3.JPG" width="200" />
+  <img src="/static/images/blog_images/2019-birthday-celebration-images/image4.JPG" width="200" />
+</p>
+
+2) The State of Debian Support
+
+Quirin Pamp spoke about the state of Debian support in the Foreman community.
+
```
Comment (rabajaj0509) [inline _posts/2019-08-28-foreman-birthday-party-at-netways.md:52]: I wanted to add few more lines here.

File: _posts/2019-08-28-foreman-birthday-party-at-netways.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+<p float="left" align='center'>
+  <img src="/static/images/blog_images/2019-birthday-celebration-images/image5.JPG" width="200" />
```
Comment (ekohl) [inline _posts/2019-08-28-foreman-birthday-party-at-netways.md]: On the pre-rendered version this appears to be 90 degrees turned.

File: _posts/2019-08-28-foreman-birthday-party-at-netways.md (hunk 1 above)
Comment (rabajaj0509) [reply _posts/2019-08-28-foreman-birthday-party-at-netways.md:52]: @quba42 feel free to add few lines here :)

Comment (quba42) [reply _posts/2019-08-28-foreman-birthday-party-at-netways.md:52]: I may get around to it tomorrow.

Comment (quba42) [reply _posts/2019-08-28-foreman-birthday-party-at-netways.md:52]: I could imagine something like the following:
```
Quirin Pamp spoke about "The State of Debian Support in Foreman/Katello".
The talk focused on Debian/Ubuntu package actions in Katello (now merged), as well as Debian/Ubuntu errata support (undergoing active review here: https://github.com/Katello/katello/pull/7961).
The latter included some discussion of the ATIX errata service to obtain the needed Debian/Ubuntu errata (https://github.com/ATIX-AG/errata_parser, https://github.com/ATIX-AG/errata_server).
```
Feel free to make adjustments as you see fit.
(Or would you rather I create a pull request against your issue branch? :stuck_out_tongue_winking_eye: )

Comment (rabajaj0509) [reply _posts/2019-08-28-foreman-birthday-party-at-netways.md:52]: Thanks for your input here :) I have added the lines mentioned by you :)

**Comment by rabajaj0509** [comment]
@ekohl @tbrisker quick merge?

File: _posts/2019-08-28-foreman-birthday-party-at-netways.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+- meetup
+- birthday
+- community
+excerpt: |
+ A quick recap of the Forman birthday event which includes new community member’s introductions, productive conversation, interesting statistics, lots of swag and cake!
+---
+
+## Prelude
+
+The Foreman birthday event was a community driven, a one-day celebration. It was held at the NETWAYS office in Nuremberg on 25th July 2019. The event majorly consisted of 2 hours of hacking and demo followed by multiple talks. Let me take you through them in more detail in this blog.
```
Comment (tbrisker) [inline _posts/2019-08-28-foreman-birthday-party-at-netways.md]:
```suggestion
The Foreman birthday event was a community driven one-day celebration. It was held at the NETWAYS office in Nuremberg on 25th July 2019. The event consisted of 2 hours of hacking and demo followed by multiple talks. Let me take you through them in more detail in this blog.
```

File: _posts/2019-08-28-foreman-birthday-party-at-netways.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+The talk focused on Debian/Ubuntu package actions in Katello (now merged), as well as Debian/Ubuntu errata support (undergoing active review [here](https://github.com/Katello/katello/pull/7961).
+The latter included some discussion of the ATIX errata service to obtain the needed [Debian/Ubuntu errata](https://github.com/ATIX-AG/errata_parser)
+
+<p float="left" align='center' class="rotate90">
+  <img src="/static/images/blog_images/2019-birthday-celebration-images/image5.JPG" width="400" height="400" />
+</p>
+
+3) A meeting with the Satellite Product Managers:
+
+For the first time at a community event, we had the Red Hat Satellite product managers, Dana Singaltery and Rich Jeriddo present to the audience. They shared the future plans for the Red Hat Satellite team and which specific domains the engineers were mostly planning to focus on. The most interesting part about this talk was an insight into how Red Hat is contributing to the wider open source community. They explained the release cycle of Red Hat Satellite to highlight the way Red Hat promotes upstream first.
```
Comment (tbrisker) [inline _posts/2019-08-28-foreman-birthday-party-at-netways.md]:
```suggestion
For the first time at a community event, we had the Red Hat Satellite product managers, Dana Singleterry and Rich Jerrido present to the audience. They shared the future plans for the Red Hat Satellite team and which specific domains the engineers were mostly planning to focus on. The most interesting part about this talk was an insight into how Red Hat is contributing to the wider open source community. They explained the release cycle of Red Hat Satellite to highlight the way Red Hat promotes upstream first.
```

File: _posts/2019-08-28-foreman-birthday-party-at-netways.md [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+
+5) The social aspect of being part of the Foreman community:
+
+[Ewoud Kohl van Wijngaarden](https://github.com/ekohl) has been a part of the Foreman community since 2012 and has a huge collection of the Foreman swags and memories which he shared with us on the birthday event.
+
+After Ewoud's talk, the party went on with interesting discussions over pizza and beer!
+
+## Swags, cake and arrangements
+
+As a return gift, all the speakers were  given the new Foreman umbrellas. For other participants, we also had t-shirts, stickers, Foreman caps and gummy bears.
```
Comment (tbrisker) [inline _posts/2019-08-28-foreman-birthday-party-at-netways.md]:
```suggestion
As a special thank-you gift, all the speakers were given a new Foreman umbrella. For other participants, we also had t-shirts, stickers, Foreman caps and gummy bears.
```

File: static/css/style.css [hunk 6]
```diff
@@ -1190,3 +1190,11 @@ code {
   text-indent: 20px;
   display: block;
 }
+
+.rotate90 {
```
Comment (tbrisker) [inline static/css/style.css]: I would much rather rotate the actual image rather than do this css hack. Let me know if you need assistance with that.

**Review by tbrisker — CHANGES_REQUESTED** [review]

File: static/css/style.css (hunk 6 above)
Comment (rabajaj0509) [reply static/css/style.css]: Yes, can you please help me with that, this looks like a hack. I tried to tweak with the image itself but due to its pixels, it seems to not be oriented correctly. Or do you suggest we can remove that picture too!

Comment (tbrisker) [reply static/css/style.css]: Fixed, pushed to your branch and also resized all of the images so they are lighter.

File: _posts/2019-08-28-foreman-birthday-party-at-netways.md [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+The talk focused on Debian/Ubuntu package actions in Katello (now merged), as well as Debian/Ubuntu errata support (undergoing active review [here](https://github.com/Katello/katello/pull/7961).
+The latter included some discussion of the ATIX errata service to obtain the needed [Debian/Ubuntu errata](https://github.com/ATIX-AG/errata_parser)
+
+<p float="left" align='center'>
+  <img src="/static/images/blog_images/2019-birthday-celebration-images/image5.JPG" width="500" />
+</p>
+
+3) A meeting with the Satellite Product Managers:
+
+For the first time at a community event, we had the Red Hat Satellite product managers, Dana Singleterry and Rich Jeriddo present to the audience. They shared the future plans for the Red Hat Satellite team and which specific domains the engineers were mostly planning to focus on. The most interesting part about this talk was an insight into how Red Hat is contributing to the wider open source community. They explained the release cycle of Red Hat Satellite to highlight the way Red Hat promotes upstream first.
```
Comment (tbrisker) [inline _posts/2019-08-28-foreman-birthday-party-at-netways.md]:
```suggestion
For the first time at a community event, we had the Red Hat Satellite product managers, Dana Singleterry and Rich Jerrido present to the audience. They shared the future plans for the Red Hat Satellite team and which specific domains the engineers were mostly planning to focus on. The most interesting part about this talk was an insight into how Red Hat is contributing to the wider open source community. They explained the release cycle of Red Hat Satellite to highlight the way Red Hat promotes upstream first.
```

**Review by tbrisker — CHANGES_REQUESTED** [review]

**Review by tbrisker — APPROVED** [review]
Thanks @rahulbajaj0509 !

**Comment by rabajaj0509** [comment]
@tbrisker thanks for all your help!

**Pull request merged**